CHANGELOG.md

```diff
@@ -4,6 +4,16 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+### 0.0.1
```

Comment:
The version bump was for `0.1.0`, so I think this entry should match...

Reply:
Good catch ... I vacillated between whether is should be minor or patch and make different choices at different times.  Will fix.